.github/workflows/terrascan.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: terrascan
+
+on:
+  workflow_call:
+    inputs:
+      iac_type:
+        required: true
+        type: string
+        default: terraform
+      policy_type:
```

Comment:
Lets add the options here, if we want to use inputs: all, aws, azure, gcp, github, k8s

Reply:
update description, no need to update worklfow if real workflow changes